pandas/_libs/tslibs/offsets.pyx

```diff
@@ -334,8 +334,6 @@ class _BaseOffset(object):
             # other is not a DateOffset object
             return False
 
-        return self._params == other._params
```

Comment:
this didn't break anything?

Reply:
Was unreachable; there was a compile-time warning about it.